sklearn/linear_model/coordinate_descent.py

```diff
@@ -766,9 +766,7 @@ def _decision_function(self, X):
         """
         check_is_fitted(self, 'n_iter_')
         if sparse.isspmatrix(X):
-            return np.ravel(safe_sparse_dot(self.coef_, X.T,
-                                            dense_output=True) +
-                            self.intercept_)
+            return safe_sparse_dot(self.coef_, X.T, dense_output=True).T + self.intercept_
```

Comment:
Hi, thanks for the PR. I actually was about to comment about the same. Not that it makes a difference but I was hoping to adopt the same transformation used in [here] (https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/base.py#L252).  Sorry for digression. Thanks!